.github/workflows/code-engine-preview.yml

```diff
@@ -3,6 +3,11 @@ name: Docker image deployment process in IBM Cloud Code Engine
 on:
   workflow_call:
     inputs:
+      #possible values: simple, platypus
+      build:
+        default: simple
+        required: false
+        type: string
```

Comment:
not sure if i am missing something but i do not see a reason for the `inputs.build`. 

both `build_push_to_cr` and `platypus_build_push_to_cr` steps look identical to me.

Reply:
We write these because this:

https://github.com/Qiskit/platypus/pull/174/files#diff-16e486445ce2ea5726b0c68b85147885aad9b8e9b6fe2dcee6f0369a8bba5eee

Reply:
ah ok.. got it. thanks for the clarification.

Reply:
Yeah, the point is that at the moment they follow the same build pattern but with the DB inclusion and `user-account` feature this will change in the very very near future.